packages/eigen/eigen.0.1.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Liang Wang <ryanrhymes@gmail.com>"
+authors: [ "Liang Wang" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/eigen"
+dev-repo: "git+https://github.com/owlbarn/eigen.git"
+bug-reports: "https://github.com/owlbarn/eigen/issues"
+doc: "https://owlbarn.github.io/eigen/eigen"
+build: [
+  ["dune" "build" "eigen_cpp/libeigen_cpp_stubs.a"]
```

Comment:
why is it not built automatically with the normal dune build command? Also is it installed with the package?

Reply:
It has to be done in stages because it compiles and external C++ library, see https://discuss.ocaml.org/t/dune-depend-on-non-dune-libraries-in-the-workspace and https://discuss.ocaml.org/t/strange-error-in-dune-building/2979 for more context

Reply:
dune is not able to decide the ordering on its own (yet)